code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,735 @@
+	randomized = FALSE
+
+
+/obj/item/organ/internal/heart/gland/heals/weak
+	name = "fleshy growth"
+	desc = "A nausea-inducing growth of flesh."
+	origin_tech = "biotech=5"
+
+/obj/item/organ/internal/heart/gland/heals/weak/activate()
+	to_chat(owner, "<span class='notice'>You feel slightly revitalized.</span>")
+	owner.adjustToxLoss(-10)
+	owner.adjustBruteLoss(-10)
+	owner.adjustOxyLoss(-10)
+	owner.adjustFireLoss(-10)
+
+/obj/item/ammo_casing/c10mm/spent
+	projectile_type = null
+
+/obj/item/ammo_casing/c45/spent
+	projectile_type = null
+
+////// MACHINERY //////
+
+/obj/machinery/computer/id_upgrader/dsbssigma
+	icon_keyboard = "laptop_key"
+	door_to_open = "dsbss_lockdown_blast"
+	upgrade_msg = "<span class='notice'>An access type was added to your ID card. You should be able to lift the lockdown with this.</span>"
+	unlock_msg = "<span class='danger'>WARNING: New user registration during emergency lockdown. Limited lockdown lift in progress.</span>"
+
+/obj/machinery/vending/medical/syndicate_access/dsbssigma
+	req_access = list(ACCESS_AWAY01)
+	products = list(/obj/item/reagent_containers/syringe = 4, /obj/item/reagent_containers/food/pill/patch/styptic = 2, /obj/item/reagent_containers/food/pill/patch/silver_sulf = 2, /obj/item/reagent_containers/applicator/brute = 1, /obj/item/reagent_containers/applicator/burn = 1,
+					/obj/item/reagent_containers/glass/bottle/charcoal = 1, /obj/item/reagent_containers/glass/bottle/salicylic = 1, /obj/item/reagent_containers/glass/bottle/saline = 1, /obj/item/reagent_containers/glass/bottle/ether = 1,
+					/obj/item/reagent_containers/glass/bottle/atropine = 1, /obj/item/reagent_containers/glass/bottle/oculine = 1, /obj/item/reagent_containers/glass/bottle/toxin = 1, /obj/item/reagent_containers/syringe/antiviral = 2,
+					/obj/item/reagent_containers/syringe/insulin = 1, /obj/item/reagent_containers/hypospray/autoinjector = 2, /obj/item/reagent_containers/food/pill/salbutamol = 2, /obj/item/reagent_containers/food/pill/mannitol = 2, /obj/item/reagent_containers/food/pill/mutadone = 1,
+					/obj/item/stack/medical/bruise_pack/advanced = 1, /obj/item/stack/medical/ointment/advanced = 1, /obj/item/stack/medical/bruise_pack = 2, /obj/item/stack/medical/splint = 2, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/reagent_containers/dropper = 1,
+					/obj/item/healthanalyzer = 1, /obj/item/healthupgrade = 1, /obj/item/reagent_containers/iv_bag/slime = 1)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/sulfonal = 1, /obj/item/reagent_containers/glass/bottle/pancuronium = 1)
+
+/obj/machinery/autolathe/hacked
+	hacked = TRUE
+
+/obj/machinery/autolathe/hacked/Initialize()
+    ..()
+    adjust_hacked(TRUE)
+
+/obj/machinery/door_control/away/button_single_use
+	var/beenused = FALSE
+	var/activation_msg = "<span class='notice'>The button briefly activates, then sparks and breaks.</span>"
+
+/obj/machinery/door_control/away/button_single_use/attack_hand(mob/user as mob)
+	..()
```

Comment:
This will call the parent code every time it's used. You could just only call it if the below condition passes.